products/jamf-pro.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Jamf Pro
+category: device
+permalink: /jamf-pro
+versionCommand: jamf-pro --version
+releasePolicyLink: https://www.jamf.com/resources/product-documentation/jamf-pro-release-notes/
+eolColumn: true
+releaseDateColumn: true
+
+releases:
+-   releaseCycle: "10.50"
+    releaseDate: 2022-10-10
+    latest: "10.50.0"
+    eol: 2025-10-10
+    link: https://learn.jamf.com/search?labelkey=product-pro-10.50.0
+
+-   releaseCycle: "10.49"
+    releaseDate: 2022-10-10
+    latest: "10.49.0"
+    eol: 2025-04-12
+    link: https://community.jamf.com/t5/jamf-pro/jamf-pro-10-49-0-beta-now-available/td-p/295325
+
+-   releaseCycle: "10.48"
+    releaseDate: 2022-10-10
+    latest: "10.48.0"
+    eol: 2024-10-12
+    link: https://community.jamf.com/t5/release-info/jamf-pro-10-48-now-available/ta-p/294694
+
+-   releaseCycle: "10.47"
+    releaseDate: 2021-04-13
+    latest: "10.47.0"
+    eol: 2024-04-13
+    link: https://community.jamf.com/t5/jamf-pro/jamf-pro-10-47-now-available/m-p/293101
+
+---
+
+> [Jamf-Pro](https://www.jamf.com/products/jamf-pro/) is a mobile device management (MDM) solution
+> that helps organizations manage and secure their Apple devices. It provides a comprehensive set
+> of features for deploying and updating apps and software, configuring device settings, enforcing
+> security policies and remotely wiping devices in the event of theft or loss.
+
+Jamf Pro releases are typically announced in the fall and released in the winter. It follows an N-3
```

Comment:
According to https://docs.jamf.com/10.28.0/jamf-pro/release-notes/Jamf_Pro_System_Requirements.html, it looks more like the support policy for Apple OS in Jamf Pro, not the Jamf Pro software support policy.

Can you provide a link that would back this information (it will also be used as the `releasePolicyLink`) ?

Reply:
Reopen this conversation: @EdwinThomaz, https://docs.jamf.com/10.28.0/jamf-pro/release-notes/Jamf_Pro_System_Requirements.html does not explain what you documented in the description. Can you provide another official link that would back the description ?

Reply:
Reopening again, https://learn.jamf.com/bundle/jamf-pro-release-notes-current/page/System_Requirements.html document the same thing as previous links. @EdwinThomaz this documentation is about the Apple OS support in Jamf Pro, it's not the Jamf Pro support policy.

Reply:
-  Adding The replacement link
-  Support policy can be viewed for each individual version

Reply:
@EdwinThomaz, I really do not see where I can confirm what you described on https://learn.jamf.com/bundle/jamf-pro-release-notes-10.50.0/page/Deprecations_and_Removals.html.

Where does it say that the previous 3 major versions are supported on https://learn.jamf.com/bundle/jamf-pro-release-notes-10.50.0/page/Deprecations_and_Removals.html or https://learn.jamf.com/bundle/jamf-pro-release-notes-current/page/System_Requirements.html ?

You also say that:
- _Jamf Pro releases are typically announced in the fall and released in the winter_
-  _The EOL date for each version of Jamf Pro is typically announced one year in advance._

Can you provide an example for each of these announcement in the comments ?